data/sql/updates/pending_db_world/haggle.sql

```diff
@@ -0,0 +1,103 @@
+--
+DELETE FROM `creature_text` WHERE `CreatureID` = 14041;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(14041, 0, 0, 'Spare some change?', 12, 0, 100, 1, 0, 0, 9088, 0, 'Haggle - Random Say'),
+(14041, 0, 1, 'I had it all!  I had it all and then I lost it.  Lost... all gone... like... my mind.  My mind is like... cheese.  I like cheese.', 12, 0, 100, 1, 0, 0, 9089, 0, 'Haggle - Random Say'),
+(14041, 0, 2, 'Forty-two... forty-two... forty-two what?  It could be anything!  Forty-two... hmmm...', 12, 0, 100, 1, 0, 0, 9098, 0, 'Haggle - Random Say'),
+(14041, 0, 3, 'Rat-kabobs!  Get your tasty Rat-kabobs here!  Get them while they last!', 12, 0, 100, 1, 0, 0, 9150, 0, 'Haggle - Random Say'),
+(14041, 0, 4, 'Rats everywhere.  Everywhere I see rats.  Always looking at me with their beedy little eyes.  I\'ll show them.  I\'ll show them all!', 12, 0, 100, 1, 0, 0, 9149, 0, 'Haggle - Random Say'),
+(14041, 1, 0, '%s yawns.', 16, 0, 100, 0, 0, 0, 9147, 0, 'Haggle - Emote 1'),
+(14041, 2, 0, '%s sifts through the trash.', 16, 0, 100, 69, 0, 0, 9141, 0, 'Haggle - Emote 2');
+
+DELETE FROM `creature` WHERE `guid` = 53788 AND `id1` = 14041;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(53788, 14041, 0, 0, 369, 0, 0, 1, 1, 0, 38.99483, 24.375648, -4.2973485, 0.366166770458221435, 190, 0, 0, 484, 0, 0, 0, 0, 0, '', 45704);
```

Comment:
I don't believe this is Haggle's spawn point. This was taken from 50664:
```
Position: X: -19.322598 Y: -17.1552 Z: -4.2140365
Orientation: 4.729842185974121093
```

Reply:
@elthehablo bench'd

Reply:
:)))))) I will take another look. Means messing with the waypoints and smart scripts again as well so that's why I postponed it

Reply:
> Means messing with the waypoints and smart scripts again as well so that's why I postponed it

I knew it would mess with your script, but I had to say it.

Reply:
CO1 or CO2, @heyitsbench ?
If CO1 we can use whichever as long as it's in the path. Hell even w/ CO2 I've seen creatures spawn a few points in and not at one end

Reply:
For pathing creatures it's difficult to get CO2 proper afaik. For example, the waypoints sent on spawn for the creatures in the first hallway event in Shattered Halls were out of bounds.

Reply:
The point he spawns in now is definitely on the path